app/views/workflow_executions/_workflow_execution.html.erb

```diff
@@ -2,17 +2,26 @@
   class="bg-white border-b dark:bg-gray-800 dark:border-gray-700"
   id="<%= dom_id(workflow_execution) %>"
 >
-  <td class="px-6 py-4 whitespace-nowrap sticky left-0 bg-white dark:bg-gray-800 z-10">
+  <td
+    class="
+      px-6
+      py-4
+      whitespace-nowrap
+      sticky
+      left-0
+      bg-white
```

Comment:
**nitpick:** Can you update the background colour on this column to be `bg-slate-50 dark:bg-slate-900`, that is how I am trying to get all primary columns that have scroll to be.

Reply:
Updated in 11361028a12f6069f80e16afbc750c6028be7147